### DISCUSSION TITLE
PlugData v0.6.0

### DISCUSSION THREAD

**Discussion by timothyschoen** [plugdata-team/plugdata#166] [General]
IMPORTANT: This version is not compatible with DAW saves from v0.5.3. If you're using a recent Actions tab build, you should be fine. Hopefully the last time that happens. But if you have DAW projects you want to keep:
 - Save all opened PlugData plugins to Pd patches
 - Remove PlugData from those projects
 - Upgrade PlugData
 - Re-add PlugData
 
For Linux users, I'd recommend using the OBS packages, or the AUR package on Arch.
OBS packages (maintained by @KottV): https://software.opensuse.org//download.html?project=home%3Aplugdata&package=plugdata
AUR packages (maintained by @aik2mlj): https://aur.archlinux.org/packages/plugdata-git

Thanks to @JoshuaACNewman, @ludnny, @kreth608 and @QuevasMz for sponsoring the development of PlugData!

This version includes many improvements, most importantly:

- Updated to ELSE-rc3, which includes new band-limited oscillators, [numbox~] and more! Now also includes the [sfont~] object, and the Live Electronics Tutorial.
- Built-in oversampling to reduce aliasing in patches with distortion or non-bandlimited synthesis. You can enable this by clicking "1x" in the statusbar.
- Installers for Windows and Mac to make the installation process faster
- Externals work in the standalone on Windows now!
- Smart-patching: Shift is the magic key
  -  Create multiple connections at once with the shift key + multiple selection.
  -  Create multiple connections in a row with the shift key + single object.
  -  Drop an object into a connection by dragging it over a connection while holding shift

Other changes:
- Fixed copy/paste activity bug
- Added implementation for [text define] object
- PlugData's Deken implementation is now much faster, and gives instant results while typing.
- Print messages are correctly concatenated
- Fixed glitches when dragging an object close to the edge of the canvas
- Faster startup time
- GraphOnParent improvements:
  - Can show name of graph
  - Can click through an empty graph, like in pd-vanilla
- [nbx] and [floatatom] Improvements:
  - Selection state works more like pd-vanilla and Max
  - Use up/down keys to increment/decrement when selected
  - Better performance
  - Fixed [nbx] size issues
  - Better dragging behaviour
- Fixed bug causing subpatcher helpfiles not being found
- Allow clicking on suggestions
- Middle-mouse pan
- Fixed various size and position inconsistencies for GUI object
- Messages now automatically open editor when created
- Can now use both delete and backspace to delete objects
- Fixed font bugs on Windows
- Fixed various problems with the [listbox] object
- Can now optionally use "else/" prefix when referring to ELSE objects
- Fixed various Windows related crashes
- Fixed bugs related to array object
- Many more bugfixes!!

Thanks to @kreth608, @60-hz, @offthesky, @Omar-Misa, @mantsar, @JoshuaACNewman, @Nek, for contributing and reporting a lot of these bugs. Sorry If I forgot anyone!

<hr /><em>This discussion was created from the release <a href='https://github.com/timothyschoen/PlugData/releases/tag/v0.6.0'>PlugData v0.6.0</a>.</em>

**Comment by enchilada404** [comment]
**Externals work in the standalone on Windows now!** 😋👍🤘
